Modules/_ssl.c

```diff
@@ -1555,7 +1555,7 @@ cipher_to_dict(const SSL_CIPHER *cipher)
     cipher_protocol = SSL_CIPHER_get_version(cipher);
     cipher_id = SSL_CIPHER_get_id(cipher);
     SSL_CIPHER_description(cipher, buf, sizeof(buf) - 1);
-    len = strlen(buf);
+    len = (int)strlen(buf);
```

Comment:
buf is 512 bytes long and always ends with a null byte, so the downcast is safe.